.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request: {}
+  workflow_dispatch: {}
+  push: 
+    branches:
+      - main
+      - master
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Nit. Midnight is such popular time to do stuff that I would rather see more automation to happen at any other time, such as:

    - cron: '42 1 * * *'

Reply:
Changed it to 4am UTC (a standard approach). Thanks for the +1.